tools/dependencies.props

```diff
@@ -2,14 +2,14 @@
   <PropertyGroup>
     <CoreFxStableVersion>4.3.0</CoreFxStableVersion>
     <RoslynVersion>2.8.0-beta4</RoslynVersion>
-    <SystemMemoryVersion>4.6.0-preview1-26614-04</SystemMemoryVersion>
-    <SystemCompilerServicesUnsafeVersion>4.6.0-preview1-26614-04</SystemCompilerServicesUnsafeVersion>
-    <SystemNumericsVectorsVersion>4.6.0-preview1-26614-04</SystemNumericsVectorsVersion>
-    <SystemBuffersVersion>4.6.0-preview1-26614-04</SystemBuffersVersion>
-    <SystemIOPipelinesVersion>4.6.0-preview1-26614-04</SystemIOPipelinesVersion>
+    <SystemMemoryVersion>4.6.0-preview1-26717-04</SystemMemoryVersion>
+    <SystemCompilerServicesUnsafeVersion>4.6.0-preview1-26908-01</SystemCompilerServicesUnsafeVersion>
+    <SystemNumericsVectorsVersion>4.6.0-preview1-26908-01</SystemNumericsVectorsVersion>
+    <SystemBuffersVersion>4.6.0-preview1-26908-01</SystemBuffersVersion>
+    <SystemIOPipelinesVersion>4.6.0-preview1-26908-01</SystemIOPipelinesVersion>
     <LibuvVersion>1.9.1</LibuvVersion>
     <TestSdkVersion>15.0.0</TestSdkVersion>
     <XunitVersion>2.2.0</XunitVersion>
-    <BenchmarkDotNetVersion>0.10.14.683</BenchmarkDotNetVersion>
+    <BenchmarkDotNetVersion>0.11.1</BenchmarkDotNetVersion>
```

Comment:
@adamsitnik, it looks like BDN doesn't support netcoreapp3.0 yet. Do you know when that will be possible? I would like to change the test projects to target netcoreapp3.0 soon.

I changed the TFM to netcoreapp3.0 and I get the following error when I try to run the tests:
https://github.com/dotnet/corefxlab/blob/59b469bcd62f8cd4209fee6c660bdaeb6d9bceb9/tests/Benchmarks/Benchmarks.csproj#L4

```text
// Build Exception:   Restoring packages for E:\GitHub\Fork\corefxlab\tests\Benchmarks\bin\Release\netcoreapp3.0\4d44f5a1-46c5-4b80-9350-6049e1a4c79a\BenchmarkDotNet.Autogenerated.csproj...
E:\GitHub\Fork\corefxlab\tests\Benchmarks\bin\Release\netcoreapp3.0\4d44f5a1-46c5-4b80-9350-6049e1a4c79a\BenchmarkDotNet.Autogenerated.csproj : error NU1201: Project Benchmarks is not compatible with netcoreapp2.1 (.NETCoreApp,Version=v2.1). Project Benchmarks supports: netcoreapp3.0 (.NETCoreApp,Version=v3.0)
  Generating MSBuild file E:\GitHub\Fork\corefxlab\tests\Benchmarks\bin\Release\netcoreapp3.0\4d44f5a1-46c5-4b80-9350-6049e1a4c79a\obj\BenchmarkDotNet.Autogenerated.csproj.nuget.g.props.
  Generating MSBuild file E:\GitHub\Fork\corefxlab\tests\Benchmarks\bin\Release\netcoreapp3.0\4d44f5a1-46c5-4b80-9350-6049e1a4c79a\obj\BenchmarkDotNet.Autogenerated.csproj.nuget.g.targets.
  Restore failed in 108.63 ms for E:\GitHub\Fork\corefxlab\tests\Benchmarks\bin\Release\netcoreapp3.0\4d44f5a1-46c5-4b80-9350-6049e1a4c79a\BenchmarkDotNet.Autogenerated.csproj.
```

Reply:
@ahsonkhan I will send a PR soon

Reply:
@ahsonkhan solved in https://github.com/dotnet/corefxlab/pull/2472